C7/UIElements/Popups/GameMenu.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Godot;
+
+public class GameMenu : Popup
+{
+
+    public GameMenu() {
+        alignment = BoxContainer.AlignMode.Center;
+        margins = new Margins(top: 100);
+    }
+
+    public override void _Ready()
+    {
+        base._Ready();
+
+        ImageTexture thisTexture = new ImageTexture();
+        Image image = new Image();
+        image.Create(370, 300, false, Image.Format.Rgb8);
+        image.Fill(Color.Color8(0, 0, 0, 0));
+        thisTexture.CreateFromImage(image);
+        this.Texture = thisTexture;
+
+        AddBackground(370, 300);
+
+        AddHeader("Main Menu", 10);
+
+        AddButton("Map", 60, "map");
+        AddButton("Load Game (Ctrl-L)", 85, "load");
+        AddButton("New Game (Ctrl-Shift-Q)", 110, "newGame");
+        AddButton("Preferences (Ctrl-P)", 135, "preferences");
+        AddButton("Retire (Ctrl-Q)", 160, "retire");
+        AddButton("Save Game (Ctrl-S)", 185, "save");
+        AddButton("Quit Game (ESC)", 210, "quit");
```

Comment:
I saw the note about italic, underline, and so forth.  I'm also wondering as we move towards Babylon, should we even note the shortcuts before they are available?  We (in some cases "I") tossed a few things in early just to add some interactivity, or the illusion thereof, but maybe we should present a more accurate picture of what's available.  At this time, I believe only ESC for Quit Game is implemented.

Reply:
from a development perspective I think it's nice to have so it's easy to hook up buttons to code once it's ready (ie. save when we can serialize a game to .SAV format), but if we don't want them for releases I can remove too